controllers/vrg_volrep_test.go

```diff
@@ -1788,14 +1788,14 @@ func pvcUnprotectedVerify(
 type pvcPreDeleteVerify func(ramendrv1alpha1.VolumeReplicationGroup, types.NamespacedName, string)
 
 func vrgPvcStatusAbsentVerify(
-	vrg ramendrv1alpha1.VolumeReplicationGroup, pvcNamespacedName types.NamespacedName, pvName string,
+	vrg ramendrv1alpha1.VolumeReplicationGroup, pvcNamespacedName types.NamespacedName, _ string,
 ) {
```

Comment:
This too

Reply:
This function must have signature:
type pvcPreDeleteVerify func(ramendrv1alpha1.VolumeReplicationGroup, types.NamespacedName, string)